sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -75,17 +75,17 @@ statement
     | ALTER TABLE tableIdentifier NOT STORED AS DIRECTORIES            #unstoreTable
     | ALTER TABLE tableIdentifier
         SET SKEWED LOCATION skewedLocationList                         #setTableSkewLocations
-    | ALTER TABLE tableIdentifier ADD (IF NOT EXISTS)?
+    | ALTER kind=TABLE tableIdentifier ADD (IF NOT EXISTS)?
```

Comment:
Minor: we don't really need kind. We can also just check for the existance of the VIEW keyword, e.g.: `ctx.VIEW != null`.